packages/blockchain/README.md

```diff
@@ -245,16 +245,18 @@ This library uses the [debug](https://github.com/visionmedia/debug) debugging ut
 
 The following initial logger is currently available:
 
-| Logger              | Description                                                 |
-| ------------------- | ----------------------------------------------------------- |
-| `blockchain:clique` | Clique operations like updating the vote and/or signer list |
+| Logger                        | Description                                                              |
+| ----------------------------- | ------------------------------------------------------------------------ |
+| `blockchain:core`             | Core blockchain operations like when a block or header is put or deleted |
+| `blockchain:consensus:clique` | Clique consensus operations like updating the vote and/or signer list    |
+| `blockchain:consensus:ethash` | Ethash consensus operations like PoW block or header validation          |
```

Comment:
Can we really please keep these logger names short (and not over-namespacify 😂 )? We do not need to be 100% consistent here and not sure if you have used loggers a lot in the past but it really becomes annoying quickly if these logger names are long.

(also this :core addition is a bit inconsistent, since we are not doing/using this on any other logger for the core class)

So let's please simply do `blockchain`, `blockchain:clique` and `blockchain:ethash`. This is mostly for our internal usage and we can make "breaking" changes here anytime.

Reply:
Changed this back to what it was before for now. Will create an issue on rethinking namespaces to allow wildcards like `DEBUG=ethjs,blockchain:*` work for all namespaces in blockchain and not unexpectedly exclude the core namespace, as is happening currently.